doc/README.md

```diff
@@ -2,27 +2,43 @@
 
 ## About Flame
 
-Flame is a modular Flutter game engine that provides a complete set of out-of-the-way solutions for games, taking advantage of the powerful infrastructure provided by Flutter but simplifying the code you need to build your projects.
+Flame is a modular Flutter game engine that provides a complete set of out-of-the-way solutions for
+games. It takes advantage of the powerful infrastructure provided by Flutter but simplifies the code
+you need to build your projects.
 
-It empowers you with a simple yet effective game loop implementation, and all the surrounding utilities you might need to use in a game: images, sprites, sprite sheets, animations, audio, input, components/object system, et cetera.
+It provides you with a simple yet effective game loop implementation, and all the surrounding
+utilities that you might need to use in a game: input, images, sprites, sprite sheets, animations,
+audio, components/object systems, collision detection, physics and so on.
```

Comment:
Flame itself does not provide physics right? The community/group provides such, but the stand alone package does not.

Reply:
I think what is referenced here is some modules available, both baked into flame and stand-alone, audio and object system (I guess it refers to oxygen) is also stand-alone. But I don't really like this sentence, so if you have any good suggestion of how to re-write it I'm open for suggestions!

Reply:
```markdown
It provides you with a simple yet effective game loop implementation, and the necessary functionalities that you might need in a game: input, images, sprites, sprite sheets, animations, collision detection and a component system that we call Flame Component System (FCS for short). 

We also provide stand-alone packages that amplify the Flame functionality:
- [flame_audio](link?) Which provides audio capabilities using the `audioplayers` package.
- [flame_forge2d](link?) Which provides physics capabilities using our own `Box2D` port called `Forge2D`.
- and later one maybe oxygen and others I forgot
```